src/data_provider/src/sysInfoMac.cpp

```diff
@@ -23,14 +23,16 @@
 #include "ports/portBSDWrapper.h"
 #include "ports/portImpl.h"
 #include "packages/packageFamilyDataAFactory.h"
-#include "packages/pkgWrapper.h"
 #include "packages/packageMac.h"
 #include "hardware/factoryHardwareFamilyCreator.h"
 #include "hardware/hardwareWrapperImplMac.h"
 #include "osPrimitivesImplMac.h"
+#include "sqlite_wrapper.h"
 
 const std::string MAC_APPS_PATH{"/Applications"};
 const std::string MAC_UTILITIES_PATH{"/Applications/Utilities"};
+const std::string MACPORTS_DB_NAME {"registry.db"};
+const std::string MACPORTS_QUERY {"SELECT state, name, version, date, location, archs FROM ports;"};
```

Comment:
Please check the rebase, because this query had a `WHERE` clause.

Reply:
Thanks @pereyra-m, It seems that I'm not mastering the art of rebases yet. Although all the commits are present It's like only the first commit has the changes.

Reply:
> It seems that I'm not mastering the art of rebases yet.

Maybe we never will !